Find a functionally equivalent code:android.view.accessibility.AccessibilityNodeInfo.makeNodeId:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * Makes a node id by shifting the <code>virtualDescendantId</code>
 * by {@link #VIRTUAL_DESCENDANT_ID_SHIFT} and taking
 * the bitwise or with the <code>accessibilityViewId</code>.
 *
 * @param accessibilityViewId A View accessibility id.
 * @param virtualDescendantId A virtual descendant id.
 * @return The node id.
 *
 * @hide
 */

@@ -1,3 +1,7 @@
 {
+    // remap it here.
+    if (virtualDescendantId == AccessibilityNodeProvider.HOST_VIEW_ID) {
+        virtualDescendantId = UNDEFINED_ITEM_ID;
+    }
     return (((long) virtualDescendantId) << VIRTUAL_DESCENDANT_ID_SHIFT) | accessibilityViewId;
 }

